projects/demo/src/modules/components/dialog/examples/3/index.ts

```diff
@@ -17,7 +18,11 @@ export class TuiDialogExampleComponent3 {
         @Inject(TuiDialogService) private readonly dialogService: TuiDialogService,
     ) {}
 
-    showDialog(content: PolymorpheusContent<TuiDialogContext>): void {
+    showDialog(
```

Comment:
Why all these changes to dialog content types?

Reply:
fixed